dockertest/output.py

```diff
@@ -767,25 +791,37 @@ def mustfail(cmdresult, failmsg=None):
     :type cmdresult: object convertible to string with variable exit_status
     :param failmsg: Additional messages for describing problem when cmd fails.
     """
+    # FIXME: temporary: backward compatibility for pre-20160330 code
+    # FIXME: remove before the next API-changing release
+    if isinstance(expected_status, basestring):
+        failmsg = expected_status
+        expected_status = 1
+
```

Comment:
Hmm, maybe this should also test if ``failmsg == None``?  Or is that being too paranoid?